CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ### Bug Fixes
 
+* oidc: If provider returns an `aud` claim as a `string` or `[]string`,
+  Boundary will properly parse the claims JSON.
+  ([cap issue](https://github.com/hashicorp/cap/issues/37),
```

Comment:
Personally I'd not specify cap issue/cap PR/PR -- I think just our normal Issue/PR links but the Issue happens to go to the cap repo and the PR is our PR pulling in the updated dep.

Reply:
ty!